pkg/upgrade/upgrade_requestor.go

```diff
@@ -161,7 +164,7 @@ func SetDefaultNodeMaintenance(opts RequestorOptions,
 
 func (m *RequestorNodeStateManagerImpl) NewNodeMaintenance(nodeName string) *maintenancev1alpha1.NodeMaintenance {
 	nm := defaultNodeMaintenance.DeepCopy()
-	nm.Name = nodeName
+	nm.Name = m.opts.NodeMaintenanceNamePrefix + "-" + nodeName
```

Comment:
nit: I prefer to use `fmt.Sprintf` but it's not a faster solution, so I won't block PR for it